package.json

```
@@ -753,10 +753,13 @@
 		"watch:desktop:esbuild": "node esbuild.mjs --watch --mode desktop",
 		"watch:desktop:tsc": "tsc --noEmit --watch --project tsconfig.app.json",
 		"test:web": "npm run build:web && npm-run-all -s test:web:*",
+		"test:web:browser": "npm run build:web && npm-run-all -s test:web:browser:*",
```

```suggestion
		"test:web:browser": "pnpm run build:web && npm-run-all -s test:web:browser:*",
```

yes I need to go change every script, I forgot